test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -4218,20 +4218,22 @@ Describe 'Invoke-WebRequest and Invoke-RestMethod support Cancellation through C
             [string]$Command = 'Invoke-WebRequest',
             [string]$Arguments = '',
             [uri]$Uri,
-            [int]$DelayMs = 100,
+            [int]$DelayMs = 1000,
```

Comment:
This will make most of the test cases take minimum 10 seconds to run since the point of the timeout is how long to wait before pressing the simulated CTRL-C. Is that acceptable to have them take that long?

Reply:
Oh, then I misunderstood the intention of this `-DelayMs` parameter. I thought it's a customized version of the `Wait-UntilTrue`.
In that case, please ignore this comment. Sorry about my confusion!